### PR TITLE
fix: cosmetic changes to context menu, fixed associated tests for Telegraf

### PR DESCRIPTION
**Pull request by daviddkkim**
Closes #

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 57 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2576)

File: src/telegrafs/components/CollectorCard.tsx [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+  ResourceCard,
+  IconFont,
+  FlexBox,
+  ConfirmationButton,
+  ButtonShape,
+  ComponentSize,
+  SquareButton,
+  Popover,
+  Appearance,
+  List,
```
Comment (hoorayimhelping) [inline src/telegrafs/components/CollectorCard.tsx:17]: good opportunity to alphabetize this list

File: src/telegrafs/components/CollectorCard.tsx [hunk 2]
```diff
@@ -82,31 +92,46 @@ class CollectorRow extends PureComponent<
...
+        <ConfirmationButton
+          color={ComponentColor.Colorless}
+          icon={IconFont.Trash_New}
+          shape={ButtonShape.Square}
+          size={ComponentSize.ExtraSmall}
+          confirmationLabel="Yes, Delete this configuration"
+          onConfirm={this.handleDeleteConfig}
+          confirmationButtonText="Confirm"
+          testID="context-delete-menu"
+        ></ConfirmationButton>
```
Comment (hoorayimhelping) [inline src/telegrafs/components/CollectorCard.tsx:108]: can this be converted into a single closing tag?

```ts
<ConfirmationButton
...
/>
```

**Review by hoorayimhelping — APPROVED** [review]

File: src/telegrafs/components/CollectorCard.tsx (hunk 2 above)
Comment (daviddkkim) [reply src/telegrafs/components/CollectorCard.tsx:108]: yup!

File: src/telegrafs/components/CollectorCard.tsx (hunk 1 above)
Comment (daviddkkim) [reply src/telegrafs/components/CollectorCard.tsx:17]: will do !

Comment (daviddkkim) [reply src/telegrafs/components/CollectorCard.tsx:14]: interesting, I made that change and pushed it up but not being refelected in the PR lol

Comment (daviddkkim) [reply src/telegrafs/components/CollectorCard.tsx:14]: Screen Shot 2021-09-15 at 11 46 40 AM

Comment (daviddkkim) [reply src/telegrafs/components/CollectorCard.tsx:14]: nvm its showing up now

**Review by TCL735 — APPROVED** [review]

**Pull request merged**